doc/classes/TextureButton.xml

```diff
@@ -29,18 +29,21 @@
 		</member>
 		<member name="texture_disabled" type="Texture2D" setter="set_texture_disabled" getter="get_texture_disabled">
 			Texture to display when the node is disabled. See [member BaseButton.disabled].
+			If not assigned, when the [TextureButton] is disabled it will display [member texture_normal] (or nothing, if [member texture_normal] itself has no texture assigned).
 		</member>
 		<member name="texture_focused" type="Texture2D" setter="set_texture_focused" getter="get_texture_focused">
-			Texture to display when the node has mouse or keyboard focus. [member texture_focused] is displayed [i]over[/i] the base texture, so a partially transparent texture should be used to ensure the base texture remains visible. A texture that represents an outline or an underline works well for this purpose. To disable the focus visual effect, assign a fully transparent texture of any size. Note that disabling the focus visual effect will harm keyboard/controller navigation usability, so this is not recommended for accessibility reasons.
+			Texture to [i]overlay on the base texture[/i] when the node has mouse or keyboard focus. Because [member texture_focused] is displayed on top of the base texture, a partially transparent texture should be used to ensure the base texture remains visible. A texture that represents an outline or an underline works well for this purpose. To disable the focus visual effect, assign a fully transparent texture of any size. Note that disabling the focus visual effect will harm keyboard/controller navigation usability, so this is not recommended for accessibility reasons.
 		</member>
 		<member name="texture_hover" type="Texture2D" setter="set_texture_hover" getter="get_texture_hover">
 			Texture to display when the mouse hovers the node.
+			If not assigned, when the [TextureButton] is hovered over it will display [member texture_normal] (or nothing, if [member texture_normal] itself has no texture assigned).
```

Comment:
```suggestion
			Texture to display when the mouse hovers the node. If not assigned, displays [member texture_normal] instead.
```

Reply:
When `texture_hover` and `texture_normal` are both assigned, and the TextureButton is hovered, then `texture_hover` is used:

https://github.com/user-attachments/assets/ecc40837-618d-47d5-a422-0b86e0548adb

When `texture_hover` is not assigned but `texture_normal` is, and the TextureButton is hovered, then `texture_normal` is used:

https://github.com/user-attachments/assets/2aff9405-4762-4b85-a842-b3536569731b

When neither `texture_hover` nor `texture_normal` are assigned, and the TextureButton is hovered, then no texture is used:

https://github.com/user-attachments/assets/7e6dfd67-78f5-4780-8d1f-5ee88ca42c2a

Reply:
```suggestion
			Texture to display when the mouse hovers over the node. If not assigned, the [TextureButton] displays [member texture_normal] instead when hovered over.
```